web/styles/app_variables.css

```diff
@@ -69,6 +69,9 @@
     --message-box-line-height: 1.214;
     --message-box-icon-width: 26px;
     --message-box-icon-height: 25px;
+    /* This width is updated with an exact pixel
+       width upon UI initialization. */
+    --message-box-timestamp-column-width: 0;
```

Comment:
Presumably this should be 42px as of this commit, since the code to set it isn't present yet?

Reply:
Made this commit leave it at 42px and then the final commit set it back to 0.